### ISSUE TITLE
Document current usage of labels for somebody-should issues

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#487]
As highlighted in issue #477, we have a lot of labels and it's not clear to newcomers (or lots of oldcomers too I expect...) what they're for.

We should rework/expand the explanation in https://github.com/DoESLiverpool/somebody-should/blob/master/README.md to explain what more of them are, provide links to some shortcut searches as examples of how to use them (e.g. a link to the closed laser-cutter issues as a quick starting point for searching through for last time we replaced the tube)

Some more hints/links to "what a good issue looks like" would also be good - using the SMART criteria mentioned in #477, etc.

It might also be that the README.md isn't the best location for that info.  It's the most logical in github terms, as it keeps it in the same repo, but I'm open to suggestions for the best place for it to live (on the wiki or in the handbook being other obvious candidates)

**Comment by aubergine10** [comment]
In addition, it would be useful to indicate which labels affect [external systems|https://github.com/DoESLiverpool/somebody-should/issues/477#issuecomment-298167191].

**Comment by aubergine10** [comment]
Git has some useful features that could help:

* [When creating an issue, show contributing guidelines](https://github.com/tiimgreen/github-cheat-sheet#contribution-guidelines) - could be used to link to useful infos about issues
* [Define default content of new issues](https://github.com/tiimgreen/github-cheat-sheet#issue_template-file) - could be used to prompt about SMART status

I'm going to try both of these things today and will report back.

Loads more stuff I didn't know about here: https://github.com/tiimgreen/github-cheat-sheet

**Comment by aubergine10** [comment]
So far so good:

![region capture 12](https://cloud.githubusercontent.com/assets/1386719/25660286/732e92d2-3003-11e7-90be-8e90c6a4b250.png)

I'll start chipping away at the labels stuff now.

**Comment by aubergine10** [comment]
What are the following:

* ESP Tech
* Room29
* Tosca

**Comment by jackie1050** [comment]
Tosca is the name of our polargraph. ESPTech relates to ESP8266 stuff.

**Comment by johnmckerrell** [comment]
Room 29 is the room off the main room that contains 4 of the permanent desks and is separately lockable (And was also the name of the room that I, @amcewen @osfameron and Ross Jones used to share before we started DoES Liverpool).

**Comment by aubergine10** [comment]
That's most of them documented: https://github.com/DoESLiverpool/somebody-should/blob/master/CONTRIBUTING.md

Tomorrow I'll try and confirm the full list of machines and their names to ensure there are relevant labels for them, and will then start linking to wiki pages, online docs (eg. vendor sites), any applicable DoES code repositories, and so on - see #488 

Unless there's anything major that I missed, this issue can probably be closed?

**Comment by stevesparrow** [comment]
Hi,

I've just seen to new labels, (Must DoES, Should DoES, Could DoES) and I'm not convinced they're any clearer than the ones they've replaced.  Are they questions?
Could we rename them to be more obvious please.  Something simple, as Guy suggested in the Milestones issue, or as simple as 'Priority: We Should'.

We've also dropped the 'Holding out for a hero', label out of the numbered system.  I think it plays a valuable role, especially in moving along several of our long term projects.  I'd like to restore it.  Is there a reason not to?
S

**Comment by aubergine10** [comment]
The label format needs to be `<number> - <caption>` to work with Huboard.

I can number `Holding out for a hero` as `6 - Holding out for a hero` to make it appear in Huboard if desired?

The other priorities are based on [this comment in previous ticket](https://github.com/DoESLiverpool/somebody-should/issues/477#issuecomment-296974055).

**Comment by stevesparrow** [comment]
Thanks Guy,

I think that would be good on the Waiting for a hero.  Happy for it to fit in language wise, it's the concept that's proved useful.  Let's hold on a short while and see if anyone else chips in on it.  Thanks.

Thanks as well for the link on the labels.  I did read that re-thread before commenting here, and joined in that conversation.  I was surprised that it was different than you'd shown in the image below.  

As I said, I think it's less obvious, and I'm not sure it's any better.   I'd like to make them more obvious, perhaps as you originally suggested in the image?  It does though feel like things are running a little hot this evening, and it could quite easily be just me who reads them that way.  Let's hold on a day or so, see any comments, and see where we are

Thanks
Steve

[](url)
![image](https://cloud.githubusercontent.com/assets/9022151/25785260/4af05ac0-3374-11e7-9bcf-9abc418d2b82.png)

**Comment by aubergine10** [comment]
Changing the text on the priority labels should be relatively easy, but it must remain in the format required by Huboard: `<number> - <string>` so perhaps something like: `1 - Priority: Must Do`?

If we drop Huboard requirement, we'd have more freedom over the format of the labels. Depends how important Huboard is to the team. Personally I think it's useful to see swim lanes of priorities (what Huboard shows) from time to time, but a decision needs to be made as to whether that's more important than clarity of what the labels mean.

**Comment by stevesparrow** [comment]
Fab, thanks.  Anything like that sounds good to me, and absolutely Huboard (I really must take a look at that!)

Thanks
Steve

**Comment by ajlennon** [comment]
Does DoES use Huboard? How do I use this?

**Comment by SayBeano** [comment]
DoES does have a huboard

https://huboard.com/DoESLiverpool/somebody-should

**Comment by aubergine10** [comment]
As discussed above, I'm planning to change the priority labels to make them a bit better. Specifically:

* 0 - Triage
* 1 - Priority: Must Do
* 2 - Priority: Should Do
* 3 - Priority: Could Do
* 4 - Won't Do
* 5 - Stale
* 6 - Waiting for a Hero

Note: Closed tickets don't appear in Huboard as far as I can tell.
